src/common/language_eeprom.cpp

```diff
@@ -0,0 +1,46 @@
+
+#include "language_eeprom.hpp"
+#include "../lang/translator.hpp"
+
+/*!
+ * Setter & getter class to store language code into a EEPROM
+ * Constructor will check if language code is not set, then it will set
+ * a default value.
+ * If it's set, then Translator is called to start translating.
+ */
+LangEEPROM::LangEEPROM()
+    : _language(0) {
+    uint16_t _language = static_cast<uint16_t>(eeprom_get_var(EEVAR_LANGUAGE).ui16);
```

Comment:
Are you sure the eeprom is properly initialized when the constructor get called? I'm sure it is not

Reply:
I am sure it is, because it's initialized first thing after eeprom class is inicialized.